DatadogRUM/Sources/RUMMonitor/Scopes/RUMUserActionScope.swift

```diff
@@ -106,7 +106,19 @@ internal class RUMUserActionScope: RUMScope, RUMContextProvider {
 
     func process(command: RUMCommand, context: DatadogContext, writer: Writer) -> Bool {
         if let expirationTime = possibleExpirationTime(currentTime: command.time), allResourcesCompletedLoading() {
-            sendActionEvent(completionTime: expirationTime, on: nil, context: context, writer: writer)
+            // Stop user action due to timeout
+            sendActionEvent(
+                completionTime: expirationTime,
+                on: RUMStopUserActionCommand(
+                    time: command.time,
+                    globalAttributes: command.globalAttributes,
+                    attributes: [:],
+                    actionType: .custom, // ignored
+                    name: nil
```

Comment:
suggestion/ It feels a bit hacky, especially given the arbitrary handling - mocking actionType, ignoring current attributes, and forwarding global unconditionally. Ideally, we shouldn't be fabricating command types within the RUM implementation - it's harmful to debugging and can obscure the real source of issues.

A cleaner alternative might be to forward the command directly:
```diff
- on: nil
+ on: command
```
Then, only consider `command.attributes` when the command type isn't `RUMStopUserActionCommand`.

WDYT?